### PR TITLE
task: create hero section component

### PR DESCRIPTION
**Pull request by niawjunior**
Desktop
image

Mobile
image

### CODE REVIEW

File: src/assets/images/CoinDesk-Logo.png
Comment (rmuti56) [inline src/assets/images/CoinDesk-Logo.png:1]: The file name should be `coin-desk-logo.png` ?

File: src/components/TheMainLayout.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
-"use client";
+"use client"
```
Comment (rmuti56) [inline src/components/TheMainLayout.tsx:1]: Look like our code format is different so we should config `prettierrc` for this project

File: src/assets/images/CoinDesk-Logo.png
Comment (niawjunior) [reply src/assets/images/CoinDesk-Logo.png:1]: Oh yesh let's me fix it

File: src/components/TheMainLayout.tsx (hunk 1 above)
Comment (niawjunior) [reply src/components/TheMainLayout.tsx:1]: Ahaa we should

File: src/assets/images/CoinDesk-Logo.png
Comment (niawjunior) [reply src/assets/images/CoinDesk-Logo.png:1]: Fixed

**Comment by rmuti56** [comment]
LGTM

**Pull request merged**